pjsip/include/pjsip-simple/blf.h

```diff
@@ -0,0 +1,254 @@
+/*
+ * Copyright (C) 2013 Maxim Kondratenko <max.kondr@gmail.com>
+  *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+#ifndef __PJSIP_SIMPLE_BLF_H__
+#define __PJSIP_SIMPLE_BLF_H__
+
+/**
+ * @file blf.h
+ * @brief SIP Extension for blf (RFC 4235)
+ */
+#include <pjsip-simple/evsub.h>
+#include <pjsip-simple/dialog-info.h>
+
+PJ_BEGIN_DECL
+
+
+/**
+ * @defgroup PJSIP_SIMPLE_BLF SIP Extension for BLF (RFC 4235)
+ * @ingroup PJSIP_SIMPLE
+ * @brief Support for SIP Extension for BLF (RFC 4235)
+ * @{
+ *
+ * This module contains the implementation of SIP Presence Extension as
+ * described in RFC 3856. It uses the SIP Event Notification framework
+ * (evsub.h) and extends the framework by implementing "presence"
+ * event package.
+ */
+
+
+/**
+ * Initialize the blf module and register it as endpoint module and
+ * package to the event subscription module.
+ *
+ * @param endpt     The endpoint instance.
+ * @param mod_evsub The event subscription module instance.
+ *
+ * @return      PJ_SUCCESS if the module is successfully
+ *          initialized and registered to both endpoint
+ *          and the event subscription module.
+ */
+PJ_DECL(pj_status_t) pjsip_blf_init_module(pjsip_endpoint *endpt,
+                        pjsip_module *mod_evsub);
+
+
+/**
+ * Get the presence module instance.
+ *
+ * @return      The presence module instance.
+ */
+PJ_DECL(pjsip_module*) pjsip_blf_instance(void);
+
+
+/**
+ * Maximum blf status info.
```

Comment:
Please add info, if this stems from RFC.

Reply:
The description is extended.